source/publishing_data/05_processing_data/processors/expression.rst

```diff
@@ -354,16 +354,19 @@ Date and time handling
      * * ``year("2014-06-07")`` returns ``2014``
        * ``hour("2014-06-07 17:00")`` returns ``17``
        * ``hour("2014-06-07 17:00", "Europe/Paris")`` returns ``19``
+       * Suppose that the current date is "2021-04-21". In this case, ``month(now())`` returns ``4``.
    * * ``add_years([datetime],[numeric])``, ``add_months([datetime],[numeric])``, ``add_days([datetime],[numeric])``, ``add_hours([datetime],[numeric])``, ``add_minutes([datetime],[numeric])``, ``add_seconds([datetime],[numeric])``
-     * Add years, months, days, hours, minutes, seconds to a datetime
-     * ``add_months("2014-11-14", "3")`` returns ``2015-02-14``
+     * * Add years, months, days, hours, minutes, seconds to a datetime
+     * * ``add_months("2014-11-14", "3")`` returns ``2015-02-14``
```

Comment:
This works, although in principle I’d rather not use quotes around numeric arguments in the doc.

Reply:
As per your suggestion, I removed such quotes from the docs. Thanks for the review!